weave/integrations/langchain/langchain.py

```diff
@@ -180,11 +180,17 @@ def _persist_run_single(self, run: Run) -> None:
                         if wv_current_run.parent_id is None:
                             use_stack = False
                         else:
-                            # Note: this is implemented as a network call - it would be much nice
-                            # to refactor `create_call` such that it could accept a parent_id instead
-                            # of an entire Parent object.
-                            parent_run = cast(
-                                Call, self.gc.get_call(wv_current_run.parent_id)
+                            # Hack in memory parent call to satisfy `create_call`
+                            # Impact is the parent won't actually represent the parent call
+                            # so we do NOT want to save the parent to the stack
+                            use_stack = False
```

Comment:
use_stack controls if the new call is added to the stack. This seems incorrect to me

Reply:
ah, controls both adding to parent children and to call stack. The intention was to not even try to construct the correct in memory repr as we will not be constructing the parent correctly in this parallel context. But we def could still add the calls to the `call_context`...